account_payment_credit_card/readme/INSTALL.rst

```diff
@@ -0,0 +1,2 @@
+
+* No specific installation required
```

Comment:
This file is not needed in this case

Reply:
@alexey-pelykh If I use [this](https://github.com/OCA/maintainer-tools#readme-generator) then automatically generate this file.

Reply:
In that case, whoever would add the feature that requires special installation will add this file - right now this file is not needed.